ceph/ceph.go

```diff
@@ -366,7 +368,7 @@ func New() *Ceph {
 	return ceph
 }
 
-// createNamespace returns namespace slice of strings composed from: vendor, class, type and ceph-daemon name
+// createNamespace returns namespace slice of strings composed from: vendor, class, type, ceph daemon name and ceph daemon id
 func createNamespace(name string) []string {
-	return []string{ns_vendor, ns_class, ns_type, name}
+	return append([]string{ns_vendor, ns_class, ns_type}, strings.Split(name, ".")...)
```

Comment:
So overall this PR looks good. Just wanted to comment on the createNamespace because I think the join in line 357 should also just be a part of this `createNamespace` function.
```go
strings.Join(createNamespace(daemonName+key), "/")
```

So instead 
```go
func createNamespace(name string) []string {
       return strings.Join(append([]string{ns_vendor, ns_class, ns_type}, strings.Split(name, ".")...), "/")
}
```
Might be more clear also if just doing something like this:
```go
prefix := []string{ns_vendor, ns_class, ns_type}
return strings.Join(append(prefix, strings.Split(name, ".")...), "/")
```